src/components/WwwFrame/TheFooter.vue

```diff
@@ -316,6 +344,20 @@ $footer-separator-color: $light-green;
 		line-height: 1.6;
 	}
 
+	ul.download-app-icons li {
+		display: inline-block;
+		height: 60px;
+	}
+
+	ul.download-app-icons img {
+		height: 100%;
+	}
+
+	ul.download-app-icons li.app-store-icon {
+		padding: 10px;
```

Comment:
In general the UI repo does most things in rem units (aside from percents). There's a handy sass function, rem-calc, that will convert px to rem for you though.
For this it would be: 
`padding: rem-calc(10);`